src/RootWithIntl.js

```diff
@@ -214,7 +208,7 @@ class RootWithIntl extends React.Component {
                         <TitledRoute
                           name="logout"
                           path="/logout"
-                          component={renderLogoutComponent(this.props.stripes)}
+                          component={renderLogoutComponent()}
```

Comment:
This is fine, but we can we simplify this even further:
```
 component={<InternalRedirect to="/" />}
```
Then we can omit `renderLogoutComponent`, which doesn't do much any longer.